server/src/main/java/org/dst/server/service/DstStringService.java

```diff
@@ -11,5 +11,10 @@
   @BrpcMeta(serviceName = "DstService", methodName = "strGet")
   DstServerProtocol.StringGetResponse strGet(DstServerProtocol.StringGetRequest request);
 
+  @BrpcMeta(serviceName = "DstService", methodName = "listPut")
```

Comment:
If we put all services in the same file, we should rename this file to `DstService`.

Reply:
Thanks. Let's do it in a separated PR.